app/buckets/routes.js

```diff
@@ -3,7 +3,7 @@ const handlers = require('./handlers');
 
 module.exports = [
   { name: 'list', pattern: '/buckets', handler: handlers.list_buckets },
-  { name: 'new', pattern: '/buckets/new', handler: handlers.new_bucket },
+  { name: 'new', pattern: '/buckets/new/:type', handler: handlers.new_bucket },
```

Comment:
I think this URL should be either `/buckets/:type/new` or `/buckets/new?type=<type>` because REST: if you were listing buckets of a type, you would expect the URL to be `/buckets/:type/` or `/buckets?type=<type>`

Reply:
👍